src/main/webapp/front-end/src/components/tasks/TasksTable.jsx

```diff
@@ -35,9 +32,47 @@ export default class TaskTable extends React.Component {
         </tr>
       </thead>
       <tbody>
-        <TaskTableItem id="1" name="Task 1" />
-        <TaskTableItem id="2" name="Task 2" />
+        { tasks ? tasks.map(o => 
+          <TasksTableItem key={o.id} data={o} {...this.props} />) : 
+          null }
       </tbody>
-    </Table>
+    </Table>;
   }
-}
\ No newline at end of file
+
+  render() {
+    return this.buildContent();
+  }
+}
+
+const mapStateToProps = state => {
+  const tasksCreated = state.getIn(['tasksCreated', 'tasks']);
+  return {
+    tasks: tasksCreated ? tasksCreated.toJS().data : null,
+    loading: state.getIn(['tasksCreated', 'loading']),
+    error: state.getIn(['tasksCreated', 'error'])
+  }
```

Comment:
not a big deal, but here could do:
`return state.getIn(['tasksCreated', 'tasks']).toJS()`
which i believe would return the exact object you are returning.  again, not hugely important.